File: DroidsAPI/src/main/java/com/github/rochedo098/droidsapi/api/types/OrganType.java
package com.github.rochedo098.droidsapi.api.types;

public enum OrganType {
    BRAIN("brain", "organic"),
    EYES("eyes", "organic"),
    HEART("heart", "organic"),
    LUNGS("lungs", "organic");

    public String status;
    private final String id;

    OrganType(String id, String status) {
        this.id = id;
        this.status = status;
    }

    public String toString() {
        return this.id;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getStatus() {
        return this.status;
    }
}

File: src/main/kotlin/com/github/rochedo098/droids/api/types/OrganType.kt
package com.github.rochedo098.droids.api.types

enum class OrganType(private val id: String, var status: String) {
    BRAIN("brain", "organic"), EYES("eyes", "organic"), HEART("heart", "organic"), LUNGS("lungs", "organic");

    override fun toString(): String {
        return id
    }
}